daccs_client/client.py

```diff
@@ -31,94 +25,96 @@ def __init__(self, fallback: Optional[bool] = True) -> None:
             if the cloud registry cannot be accessed, defaults to True
         :type fallback: Optional[bool], optional
         :raises requests.exceptions.RequestException: Raised when there is an issue
-            conencting to the cloud registry and `fallback` is False
-        :raises UserWarning: Raised when there is an issue conencting to the cloud registry
+            connecting to the cloud registry and `fallback` is False
+        :raises UserWarning: Raised when there is an issue connecting to the cloud registry
             and `fallback` is True
         :raise RuntimeError: If cached registry needs to be read but there is no cache
         """
         self._fallback = fallback
-        self._using_fallback = False
         self._nodes: dict[str, DACCSNode] = {}
+        self._registry: dict = {}
 
         try:
-            self._registry = requests.get(REG_URL)
-            self._registry.raise_for_status()
-        except (requests.exceptions.RequestException, requests.exceptions.ConnectionError) as e:
-            if not self._fallback:
-                raise
-            else:
-                self._using_fallback = True
+            registry = requests.get(NODE_REGISTRY_URL)
+            registry.raise_for_status()
+        except (requests.exceptions.RequestException, requests.exceptions.ConnectionError):
+            if self._fallback:
                 warnings.warn("Cannot retrieve cloud registry. Falling back to cached version")
-
-        if not self._using_fallback:
```

Comment:
If you're doing if/else statements, I find it way more readable to do:

```python
if ...:
   ...
else:
   ...

```

and just swap the bodies of each block instead of:

```python
if not ...:
   ...
else:
   ...

```